refinery/tool_manager/utils.py

```diff
@@ -11,3 +17,132 @@ class AdminFieldPopulator(admin.ModelAdmin):
     def __init__(self, model, admin_site):
         super(AdminFieldPopulator, self).__init__(model, admin_site)
         self.list_display = [field.name for field in model._meta.fields]
+
+
+def create_tool_definition_from_workflow(workflow_data):
```

Comment:
We will probably need some other method like `create_tool_definition_from_vis_tool()`